internal/system/agent/services.go

```diff
@@ -25,35 +29,349 @@ const (
+					StopDockerContainer(edgexSupportLogging)
+				}
+				break
+
+			case edgexConfigSeed:
+				if Conf.OsLevelOperations {
+					OsExecuteStop(edgexConfigSeed, params[0])
+				} else if Conf.DockerLevelOperations {
+					StopDockerContainer(edgexConfigSeed)
+				}
+				break
+
+			default:
+				LoggingClient.Info(fmt.Sprintf(">> Unknown service: %v", service))
+				break
+			}
+
+		case START:
+			switch service {
+			case edgexSupportNotifications:
+				StartDockerContainerCompose(edgexSupportNotifications)
+				break
+
+			case edgexCoreData:
+				StartDockerContainerCompose(edgexCoreData)
+				break
+
+			case edgexCoreMetadata:
+				StartDockerContainerCompose(edgexCoreMetadata)
+				break
+
+			case edgexCoreCommand:
+				StartDockerContainerCompose(edgexCoreCommand)
+				break
+
+			case edgexExportClient:
+				StartDockerContainerCompose(edgexExportClient)
+				break
+
+			case edgexExportDistro:
+				StartDockerContainerCompose(edgexExportDistro)
+				break
+
+			case edgexSupportLogging:
+				StartDockerContainerCompose(edgexSupportLogging)
+				break
+
+			case edgexConfigSeed:
+				StartDockerContainerCompose(edgexConfigSeed)
+				break
+
+			default:
+				LoggingClient.Info(fmt.Sprintf(">> Unknown service: %v", service))
+				break
+			}
+
+		case RESTART:
+			switch service {
+			case edgexSupportNotifications:
+				StopAndStartDockerContainer(edgexSupportNotifications, params[0])
+				break
+
+			case edgexCoreData:
+				StopAndStartDockerContainer(edgexCoreData, params[0])
+				break
+
+			case edgexCoreCommand:
+				StopAndStartDockerContainer(edgexCoreCommand, params[0])
+				break
+
+			case edgexCoreMetadata:
+				StopAndStartDockerContainer(edgexCoreMetadata, params[0])
+				break
+
+			case edgexExportClient:
+				StopAndStartDockerContainer(edgexExportClient, params[0])
+				break
+
+			case edgexExportDistro:
+				StopAndStartDockerContainer(edgexExportDistro, params[0])
+				break
+
+			case edgexSupportLogging:
+				StopAndStartDockerContainer(edgexSupportLogging, params[0])
+				break
+
+			case edgexConfigSeed:
+				StopAndStartDockerContainer(edgexConfigSeed, params[0])
+				break
+
+			default:
+				LoggingClient.Info(fmt.Sprintf(">> Unknown service: %v", service))
+				break
+			}
+		}
 	}
 	return true
 }
 
-func getConfig(services []string) bool {
+func processResponse(resp *http.Response) ConfigRespMap {
+	var crp = ConfigRespMap{}
+	buf := new(bytes.Buffer)
+	if buf != nil {
+		buf.ReadFrom(resp.Body)
+		responseJSON = buf.String()
+
+		nsm := make(map[string]interface{})
+
+		err := json.Unmarshal([]byte(responseJSON), &nsm)
+		if err != nil {
+			LoggingClient.Error(fmt.Sprintf("ERROR: {%v}", err))
+		}
+
+		json.Unmarshal([]byte(responseJSON), &crp)
+
+		crp.Config = make(map[string]interface{})
+		json.Unmarshal([]byte(responseJSON), &crp.Config)
+	}
+	return crp
+}
+
+func getConfig(services []string) (ConfigRespMap, error) {
+
+	const (
+		configApiPath = "/api/v1/config"
+	)
```

Comment:
This constant already exists. Please don't redefine. `internal/constants.go`

Reply:
Done.